Client/Frontend/Home/FxHomeJumpBackInViewModel.swift

```diff
@@ -41,41 +45,67 @@ class FirefoxHomeJumpBackInViewModel: FeatureFlagsProtocol {
         self.tabManager = tabManager
     }
 
-    var layoutVariables: JumpBackInLayoutVariables {
-        let horizontalVariables = JumpBackInLayoutVariables(columns: 2, scrollDirection: .horizontal, maxItemsToDisplay: 4)
-        let verticalVariables = JumpBackInLayoutVariables(columns: 1, scrollDirection: .vertical, maxItemsToDisplay: 2)
+    // The maximum number of items to display in the whole section
+    static var maxItemsToDisplay: Int {
+        if deviceIsiPad {
+            return 3 // iPad
+        } else if deviceIsInLandscapeMode {
+            return 4 // iPhone in landscape
+        } else {
+            return 2 // iPhone in portrait
+        }
+    }
 
-        let deviceIsiPad = UIDevice.current.userInterfaceIdiom == .pad
-        let deviceIsInLandscapeMode = UIWindow.isLandscape
-        let horizontalSizeClassIsCompact = UIScreen.main.traitCollection.horizontalSizeClass == .compact
+    // The maximum number of items a column can contain
+    static var maxNumberOfItemsInColumn: Int {
+        return deviceIsiPad ? 1 : 2
+    }
```

Comment:
nit: we should improve the comment statement or not add it as this var is very self explanatory that it gives `max number of Items In Column` as when `device is iPad` we get 1 otherwise 2.

Reply:
I wasn't sure to put a comment on this one, but then having `numberOfItemsInColumn` and `maxNumberOfItemsInColumn` could be confusing on how it's used? So I put a comment because of that.